octavia_f5/controller/worker/controller_worker.py

```diff
@@ -481,6 +484,11 @@ def ensure_amphora_exists(self, load_balancer_id):
                 load_balancer_id=load_balancer_id,
                 compute_flavor=CONF.host,
                 status=lib_consts.ACTIVE)
+        if device_amp.compute_flavor != CONF.host:
+            self._amphora_repo.update(
+                db_apis.get_session(),
+                device_amp.id,
+                **{'compute_flavor': CONF.host})
```

Comment:
Please use the more idiomatic syntax `compute_flavor=CONF.host` instead of `**{...}`.